CompositeFrame/ServiceFrame-2017.ix

```diff
@@ -31,4 +31,8 @@ xmlelement ServiceFrame {
 	&include ServiceFrame/journeyPatterns.ix
 
 	&include ServiceFrame/timeDemandTypes.ix
+	
+	&include ServiceFrame/notices.ix
+	
+	&include ServiceFrame/NoticesAssignment.ix
```

Comment:
Could you rename NoticesAssignment into noticeAssignments.ix? If it is a hassle, I can do it for you.